test/basic.html

```diff
@@ -73,6 +73,24 @@
           done();
         }, 12);
       });
+      
+      test('toast fires closed event', function(done) {
+        toast = fixture('basic');
+        toast.duration = 10;
```

Comment:
here the duration is so short that the toast doesn't have the time to complete the transition, hence is never opened, and as a consequence it won't be closed.
You can update these tests with something like this:
```javascript
      test('toast fires opened event', function(done) {
        toast = fixture('show');
        toast.addEventListener('iron-overlay-opened', function() {
          done();
        });
      });

      test('toast fires closed event', function(done) {
        toast = fixture('show');
        toast.addEventListener('iron-overlay-opened', toast.close.bind(toast));
        toast.addEventListener('iron-overlay-closed', function() {
          done();
        });
      });